### PR TITLE
refact(doc)update readme

### PR DESCRIPTION
**Pull request by gprasath**
Signed-off-by: gprasath <giridhara.prasad@mayadata.io>

**What this PR does / why we need it**:
- Updated readme and contributing.md by keeping only e2e-tests specific info.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Checklist**

* [ ] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [ ] Have you added task comments where necessary? 
* [ ] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
-There are several areas of Litmus that could use your help. For starters, you could help in improving the sections in this document by either creating a new issue describing the improvement or submitting a pull request to this repository. 
+There are several areas of e2e that could use your help. For starters, you could help in improving the sections in this document by either creating a new issue describing the improvement or submitting a pull request to this repository. 
 
 * If you are a first-time contributor, please see [Steps to Contribute](#steps-to-contribute).
-* If you would like to suggest new tests to be added to litmus, please go ahead and [create a new issue](https://github.com/openebs/litmus/issues/new) describing your test. All you need to do is specify the workload type and the operations that you would like to perform on the workload.
+* If you would like to suggest new tests to be added to this repo, please go ahead and [create a new issue](https://github.com/openebs/litmus/issues/new) describing your test. All you need to do is specify the workload type and the operations that you would like to perform on the workload.
```
Comment (nsathyaseelan) [inline CONTRIBUTING.md:8]: @giri change the repo URL to `https://github.com/openebs/e2e-tests/issues/new`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 * If you would like to work on something more involved, please connect with the OpenEBS Contributors. See [OpenEBS Community](https://github.com/openebs/openebs/tree/master/community)
 * If you would like to make code contributions, all your commits should be signed with Developer Certificate of Origin. See [Sign your work](#sign-your-work). 
 
 ## Steps to Contribute
 
-* Find an issue to work on or create a new issue. The issues are maintained at [openebs/litmus](https://github.com/openebs/litmus/issues). You can pick up from a list of [good-first-issues](https://github.com/openebs/litmus/labels/good%20first%20issue).
+* Find an issue to work on or create a new issue. The issues are maintained at [openebs/e2e-tests](https://github.com/openebs/e2e-tests/issues). You can pick up from a list of [good-first-issues](https://github.com/openebs/litmus/labels/good%20first%20issue).
```
Comment (nsathyaseelan) [inline CONTRIBUTING.md:14]: Here also need to change the repo URL:- `https://github.com/openebs/e2e-tests/labels/good%20first%20issue`

File: README.md [hunk 3]
```diff
@@ -4,68 +4,67 @@
...
 And as a developer friendly framework, it also provides helpful playbooks to quickly spin-up Kubernetes clusters on different 
 cloud & on-premise platforms on which to run the experiments! 
 
-For details on the architecture, implementation & reference usecases, please read the [litmus docs](https://docs.litmuschaos.io)
-
 ## Getting Started
 
-Litmus experiment jobs(also called Litmusbooks) run using a dedicated ServiceAccount in the Litmus namespace. Setup RBAC & custom
-resource definitions (CRDs) via kubectl or helm, as shown below: 
+The test experiment jobs(also called Litmusbooks) run using a dedicated ServiceAccount in the Litmus namespace. Setup RBAC & custom
```
Comment (nsathyaseelan) [inline README.md:34]: `also called Litmusbooks` is this neede?

Comment (nsathyaseelan) [inline README.md:34]: `also called Litmusbooks` is this needed?

File: CONTRIBUTING.md (hunk 2 above)
Comment (nsathyaseelan) [inline CONTRIBUTING.md]: Here also need to change the repo URL:- `https://github.com/openebs/e2e-tests/labels/good%20first%20issue`

File: CONTRIBUTING.md (hunk 1 above)
Comment (nsathyaseelan) [inline CONTRIBUTING.md]: @giri change the repo URL to `https://github.com/openebs/e2e-tests/issues/new`

**Review by nsathyaseelan — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (gprasath) [reply README.md:34]: yes

**Review by nsathyaseelan — APPROVED** [review]
lgtm

**Pull request merged**